dashboard/test/helpers/application_helper_test.rb

```diff
@@ -83,18 +83,19 @@ def request
     assert(browser.version.to_s.to_i == 34)
   end
 
-  test 'certificate images for hoc-type scripts are all hoc certificates' do
-    # old hoc, new hoc, frozen, flappy, playlab, and starwars are all the same certificate
+  test 'script_certificate_image_url helper encodes correct information' do
     user = create :user
-    hoc_2013 = Script.get_from_cache(Script::HOC_2013_NAME)
-    assert_equal script_certificate_image_url(user, hoc_2013), script_certificate_image_url(user, Script.get_from_cache(Script::HOC_NAME))
-    assert_equal script_certificate_image_url(user, hoc_2013), script_certificate_image_url(user, Script.get_from_cache(Script::FROZEN_NAME))
-    assert_equal script_certificate_image_url(user, hoc_2013), script_certificate_image_url(user, Script.get_from_cache(Script::FLAPPY_NAME))
-    assert_equal script_certificate_image_url(user, hoc_2013), script_certificate_image_url(user, Script.get_from_cache(Script::PLAYLAB_NAME))
-    assert_equal script_certificate_image_url(user, hoc_2013), script_certificate_image_url(user, Script.get_from_cache(Script::STARWARS_NAME))
-
-     # but course1 is a different certificate
-    assert_not_equal script_certificate_image_url(user, Script.get_from_cache(Script::HOC_2013_NAME)), script_certificate_image_url(user, Script.get_from_cache(Script::COURSE1_NAME))
+    assert_certificate_url_encodes user, Script.get_from_cache(Script::HOC_2013_NAME)
+    assert_certificate_url_encodes user, Script.get_from_cache(Script::HOC_NAME)
+    assert_certificate_url_encodes user, Script.get_from_cache(Script::FROZEN_NAME)
+    assert_certificate_url_encodes user, Script.get_from_cache(Script::FLAPPY_NAME)
+    assert_certificate_url_encodes user, Script.get_from_cache(Script::PLAYLAB_NAME)
+    assert_certificate_url_encodes user, Script.get_from_cache(Script::STARWARS_NAME)
+    assert_certificate_url_encodes user, Script.get_from_cache(Script::COURSE1_NAME)
+    assert_certificate_url_encodes user, Script.get_from_cache(Script::COURSE2_NAME)
+    assert_certificate_url_encodes user, Script.get_from_cache(Script::COURSE3_NAME)
+    assert_certificate_url_encodes user, Script.get_from_cache(Script::COURSE4_NAME)
+    assert_certificate_url_encodes user, Script.get_from_cache(Script::MINECRAFT_NAME)
```

Comment:
I feel like there's a way to enumerate through all of the script names here so we don't have to change this every time there's a new script